src/components/Sections/SimpleMode/SimpleModeList.js

```diff
@@ -1,36 +1,40 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import PropTypes from 'prop-types';
-import { withTranslation } from 'react-i18next';
+import { useTranslation } from 'react-i18next';
 import { Image, List } from 'semantic-ui-react';
 import { useSelector } from 'react-redux';
 
 import WipErr from '../../shared/WipErr/WipErr';
 import { FrownSplash } from '../../shared/Splash/Splash';
-import { isEmpty } from '../../../helpers/utils';
+import { isEmpty, isToday, noop } from '../../../helpers/utils';
 import { SectionLogo } from '../../../helpers/images';
 import {
   mdbGetDenormCollectionWUnitsSelector,
   mdbGetDenormContentUnitSelector,
   settingsGetUILangSelector
 } from '../../../redux/selectors';
 import { useSimpleModeQuery } from '../../../redux/api/simpleMode';
+import { useInterval } from '../../../helpers/timer';
 
-const SimpleModeList = ({ filesLanguages, t, renderUnit, selectedDate }) => {
-
+const SimpleModeList = ({ filesLanguages, renderUnit, selectedDate }) => {
+  const { t }      = useTranslation();
   const uiLanguage = useSelector(settingsGetUILangSelector);
 
-  const { isError, isLoading, isSuccess, error, data } = useSimpleModeQuery({
+  const { isError, isLoading, isSuccess, error, data, refetch } = useSimpleModeQuery({
     date            : selectedDate,
     uiLanguage,
     contentLanguages: filesLanguages
   });
 
+  const callback = useCallback(() => isToday(selectedDate) ? refetch : noop, [selectedDate, refetch]);
+  useInterval(callback, 60 * 1000);
```

Comment:
What that fixes?
Why is that needed?

Reply:
Imagine: you're looking on a download page. Then some part of lesson is over. Now you should F5 until it will be available to download. This code eliminates it by trying to refetch data every minute

Reply:
Nice addition.